.editorconfig

```diff
@@ -163,6 +163,11 @@ dotnet_diagnostic.IDE0044.severity = warning
 # RS0016: Only enable if API files are present
 dotnet_public_api_analyzer.require_api_files = true
 
+# PROTOTYPE(list-patterns) https://github.com/dotnet/roslyn/issues/50357
+dotnet_diagnostic.RS0016.severity = none
+dotnet_diagnostic.RS0017.severity = none
+dotnet_diagnostic.RS0027.severity = none
```

Comment:
@jcouv This does not seem to work..

Reply:
Ouch, sorry to hear :-/
We may have to debug to see why the suppression isn't working.

@mavasani Would you have any clue what we may be doing wrong?
I see other suppressions for analyzers in the file (such as `dotnet_diagnostic.RS2008.severity = none`). So I would have expected this to work.

Reply:
Debugged and confirmed the issue. Filed https://github.com/dotnet/roslyn/issues/52014

I was able to suppress RS0016 in source-generated syntax trees via `Settings.props` by modifying this section:
```
        <NoWarn>$(NoWarn);1573;1591;1701;RS0016</NoWarn>
```